opm/parser/eclipse/Deck/Section.hpp

```diff
@@ -47,7 +47,7 @@ class Section : public DeckView {
 
         // returns whether the deck has all mandatory sections and if all sections are in
         // the right order
-        static bool checkSectionTopology(const Deck& deck,
```

Comment:
The getMessageContainer() method must be const. The compiler should not complain about that if the message container member is mutable (if not, you would not be allowed to return the non-const reference in a const method).